public-api/src/test/java/org/pmiops/workbench/cdr/DataBrowserControllerTest.java

```diff
@@ -435,36 +435,36 @@ public void testConceptSearchMaxResults() throws Exception{
         ResponseEntity<ConceptListResponse> response = dataBrowserController.searchConcepts(new SearchConceptsRequest()
                 .domain(Domain.CONDITION)
                 .maxResults(1)
-                .minCount(0));
+                .minCount(0), "test/ehr/condition");
       assertThat(response.getBody().getItems()).containsExactly(CLIENT_CONCEPT_5);
     }
 
     @Test
     public void testCountConceptSearchEmptyQuery() throws Exception{
         ResponseEntity<ConceptListResponse> response = dataBrowserController.searchConcepts(new SearchConceptsRequest()
-                .domain(Domain.CONDITION));
+                .domain(Domain.CONDITION),"test/ehr/condition");
         assertThat(response.getBody().getItems()).containsExactly(CLIENT_CONCEPT_1, CLIENT_CONCEPT_5, CLIENT_CONCEPT_6);
     }
 
     @Test
     public void testConceptSearchStandardConcept() throws Exception{
         ResponseEntity<ConceptListResponse> response = dataBrowserController.searchConcepts(new SearchConceptsRequest().query("002")
-                .standardConceptFilter(StandardConceptFilter.STANDARD_CONCEPTS));
+                .standardConceptFilter(StandardConceptFilter.STANDARD_CONCEPTS),"test/ehr/condition");
```

Comment:
nit: space between parameters (we should have a linter for this)

Reply:
Added. Yes linter in space would help. Not sure how to add it though.